vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -178,5 +189,16 @@ def parse_base_item(item)
         :resource_version   => item.metadata.resourceVersion
       }
     end
+
+    HUMAN_SIZE_SUFFIXES = %w(Ki Mi Gi Ti)
+
+    def human_to_value(value)
+      exp_index = HUMAN_SIZE_SUFFIXES.index(value[-2..-1])
+      if exp_index.nil?
+        return Integer(value)
+      else
+        return Integer(value[0..-3]) * 1024**(exp_index + 1)
```

Comment:
Ok, I still think there is something for this in ActiveSupport, but maybe I'm confused.  @tenderlove Do you know?